corehq/motech/fhir/utils.py

```diff
@@ -79,12 +79,23 @@ def load_fhir_resource_types(fhir_version=FHIR_VERSION_4_0_1, exclude_resource_t
     return resource_types
 
 
-def require_fhir_json_accept_headers(view_func):
+def validate_accept_header_and_format_param(view_func):
     @wraps(view_func)
     def _inner(request, *args, **kwargs):
-        accept_header = request.META.get('HTTP_ACCEPT')
-        if accept_header and accept_header not in HQ_ACCEPTABLE_FHIR_MIME_TYPES + ['*/*']:
-            return JsonResponse(status=406, data={'message': "Not Acceptable"})
+        def _get_format_param(request):
```

Comment:
Does FHIR not support Patch? I wonder if it would be useful to add now even if we don't use it right away.

Reply:
FHIR allows for PATCH: https://hl7.org/fhir/http.html#patch

But I don't think it is _required_ for us to support it. Just as FHIR _allows_ for requests and responses to be formatted [in RDF format](https://www.hl7.org/fhir/rdf.html#example), I don't think it is a _requirement_ that a FHIR API must support it.

I like the idea of PATCH, but my instinct is to wait until it's needed before we implement it.